interface/client/lib/helpers/templateHelpers.js

```diff
@@ -30,12 +30,49 @@ Template.registerHelper('CurrentBlock', function(){
 });
 
 /**
-Return the current dirname.
+Return the preload modules dirname.
 
 @method (dirname)
 **/
-Template.registerHelper('dirname', function(){
-    return window.dirname;
+Template.registerHelper('preload_dirname', function(){
+    return window.mist.dirname + '/modules/preload';
+});
+
+/**
+Return the Mist API.
+
+@method (mist)
+**/
+Template.registerHelper('mist', function(){
+    return window.mist;
+});
+
+
+/**
+Return the app mode.
+
+@method (mode)
+**/
+Template.registerHelper('mode', function(){
+    return window.mist.mode;
+});
+
+/**
+Return the friendly app name.
+
+@method (appName)
+**/
+Template.registerHelper('appName', function(){
+    return window.mist.mode === 'mist' ? 'Mist' : 'Ethereum Wallet';
+});
+
+/**
+Return the app icon path.
+
+@method (iconPath)
+**/
+Template.registerHelper('appIconPath', function(){
+    return 'file://'+ window.mist.dirname +'/icons/'+ window.mist.mode +'/icon2x.png';
```

Comment:
I changed things. I added in more global template helpers that made sense - and then removed the local equivalents from the "about" template. Because I needed the same ones in the "updateAvailable" template.